website/pages/en/help.js

```diff
@@ -34,19 +34,23 @@ function Help(props) {
 
           <strong>Email Us</strong>
           <p>
-            We have a few ways that you can contact us via email. You can{" "}
-            <a href="mailto:hubs-feedback@mozilla.com">
+            You can contact us via {" "}
+            <a href="mailto:info@hubsfoundation.org">
+              email
+            </a>{" "}. 
+            {/* You can{" "}
+            <a href="mailto:info@hubsfoundation.org">
               get subscription support,
             </a>{" "}
             ask{" "}
-            <a href="mailto:hubs-support@mozilla.com">
+            <a href="mailto:info@hubsfoundation.org">
               questions about hubs cloud,
             </a>{" "}
             or contact us about{" "}
-            <a href="mailto:enterprise-hubs@mozilla.com">
+            <a href="mailto:info@hubsfoundation.org">
```

Comment:
I think this should be base64 encoded.
```suggestion
            <a href="mailto:%69%6e%66%6f%40%68%75%62%73%66%6f%75%6e%64%61%74%69%6f%6e%2e%6f%72%67">
```